test/quantization/test_quantized_op.py

```diff
@@ -1969,9 +1942,9 @@ def test_instance_norm(self):
                 self.assertTrue(pct_diff_off_by_one < 0.01)
 
     @skipIfNoFBGEMM
-    def test_batch_norm2d_relu(self):
+    def test_batch_norm_1d_2d_3d_relu(self):
```

Comment:
nit: maybe just test_batch_norm_relu and add dimensions in the docstring if needed.

Reply:
sure, sounds good